### ISSUE TITLE
Next release (0.0.12?)

### ISSUE THREAD

**Issue by axic** [nodejs/uvwasi#166]
It would be nice to have a new release out, given the various merged fixes, and that the last release came out a year ago.

Is there anyone around to make this happen?

**Comment by cjihrig** [comment]
I'll take care of this.

**Comment by cjihrig** [comment]
@axic this is only blocked on getting reviews for two one line changes: https://github.com/nodejs/uvwasi/pull/167 and https://github.com/nodejs/uvwasi/pull/168

**Comment by axic** [comment]
Thank you!
